src/GitHub.App/ViewModels/PullRequestFileViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace GitHub.ViewModels
+{
+    public class PullRequestFileViewModel : IPullRequestFileViewModel
```

Comment:
Well to me the model is the model you get from the backend (`PullRequestFile` currently). This is transformed into a tree at the viewmodel layer for view purposes, making it a view model :)